Resources/Prototypes/Entities/Objects/Weapons/Guns/Pistols/pistols.yml

```diff
@@ -198,7 +209,7 @@
   name: N1984
   parent: BaseWeaponPistol
   id: WeaponPistolN1984 # the spaces in description are for formatting.
-  description: The sidearm of any self respecting officer.     Comes in .45 magnum, the lord's caliber. 
+  description: The sidearm of any self respecting officer.     Comes in .45 magnum, the lord's caliber.
```

Comment:
:O

Reply:
Lol, end of line removals. Ignore it, this is supposed to happen.